src/theory/ff/sub_theory.h

```diff
@@ -103,7 +103,14 @@ class SubTheory : protected EnvObj, protected context::ContextNotifyObj
   // Uninitialize the polynomial ring, clear post-registration data structures.
   void clearPolyRing();
 
+  // Ensure we have at least `numDisequalities` spare inverses.
+  //
+  // If more inverses are added, this will call clearPolyRing.
```

Comment:
We use `/** ..... */` style of comments for header documentation.

Reply:
Big oof. Yay for vim macros :)